migration/src/lib/akumina-site.js

```diff
@@ -53,15 +54,31 @@ class AkuminaSite {
   /**
    * Import files to SharePoint library
    *
-   * @param {string} listName SharePoint list name
+   * @param {string} listPath Path to SharePoint list
    * @param {Array<object>} items An array of items to import
-   * @param {string} items[n].filePath Path to the file to upload
-   * @param {object} items[n].metadata Metadata to be added to the file
+   * @param {string} items[n].path Path to the file to upload
+   * @param {string} items[n].name Name of file being uploaded
+   * @param {object} items[n].data Metadata to be added to the file
    */
-  async importFiles(listName, items, batchSize = 100) {
-    // Loop through the files provided
-    // Upload each item
-    // Update each uploaded item with the metadata
+  async importFiles(listPath, items) {
+    try {
+      // Loop through files provided
+      for (let i = 0; i < items.length; i++) {
+        const data = await fs.readFile(items[i].path, { encoding: null });
```

Comment:
Can we just not name this data? It's confusing with also having `items[i].data`. Maybe `file` or `fileContents`?